### DISCUSSION TITLE
River heights

### DISCUSSION THREAD

**Discussion by Merhindra** [trickypr/bom-weather-docs#10] [Q&A]
Do you know if we are able to pull river height and the min / mod / major flood heights? 

The data is available on the web site so hoping its possible

**Comment by trickypr** [comment]
I am still waiting on BoM's reply to my email surrounding copyright, so I am not able to help you at the moment. Sorry

**Comment by Merhindra** [comment]
No problem. I have reached out to them as well so hopefully the more people
who ask might speed it up
